packaging/fedora/snap-mgmt.sh

```diff
@@ -89,6 +89,7 @@ purge() {
     # opportunistic as those might not be actually mounted
     for mnt in /run/snapd/ns/*.mnt; do
         umount -l "$mnt" || true
+        rm -f t "$mnt"
```

Comment:
Is there an extra `t` here ? or is my eye playing tricks?

Reply:
ouch! Thanks for catching this. In the x11 session my keyboard was playing tricks on me.